README.md

```diff
@@ -215,6 +215,40 @@ app.get('/.well-known/acme-challenge/:acmeToken', function(req, res, next) {
 });
 ```
 
+### PHP Apps
+
+Add the following to `.well-known/acme-challenge/index.php`
+
+```php
+<?php
+$request = $_SERVER['REQUEST_URI'];
+if(preg_match('#^/.well-known/acme-challenge/#', $request) === 0) {
+    return;
+}
+
+if(isset($_ENV['ACME_KEY'])) die(trim($_ENV['ACME_KEY']));
+
+foreach($_ENV as $key => $value) {
+    if(preg_match('#^ACME_TOKEN_#', $key)) {
+        $number = str_replace('ACME_TOKEN_', '', $key);
+        die(trim($_ENV['ACME_KEY_'.$number]));
```

Comment:
Not exactly.

`acme-challenge/{ACME_TOKEN_x}` would serve `ACME_KEY_x` as it's content.

There are many other examples in the README. They are fairly simple, and should be understandable even if you don't know the language they're implemented in.
https://github.com/dmathieu/sabayon#configuring-your-application